nautobot/core/settings.yaml

```
@@ -0,0 +1,1657 @@
+
+      The following example gives a user a reasonable amount of access to add devices to a single location
+      (Building HQ in this case):
+
+      ```python
+      {
+          'dcim.add_device': {'location__name__in':  ['HQ'], 'location__location_type__name__in': ['Building']},
+          'dcim.view_device': {'location__name__in':  ['HQ'], 'location__location_type__name__in': ['Building']},
+          'dcim.view_devicetype': None,
+          'extras.view_role': None,
+          'extras.view_status': None,
+          'dcim.view_location': {'name__in':  ['HQ'], 'location_type__name__in': ['Building']},
+          'dcim.view_manufacturer': None,
+          'dcim.view_region': None,
+          'dcim.view_rack': None,
+          'dcim.view_rackgroup': None,
+          'dcim.view_platform': None,
+          'virtualization.view_cluster': None,
+          'virtualization.view_clustergroup': None,
+          'tenancy.view_tenant': None,
+          'tenancy.view_tenantgroup': None,
+      }
+      ```
+    "properties":
+      "additionalProperties": true
+    "see_also":
+      "Guide to object permissions": "../guides/permissions.md"
+    "type": "object"
+  "FORCE_SCRIPT_NAME":
+    "default": ~
+    "description": >-
+      If not None, this will be used as the value of the SCRIPT_NAME environment variable in any HTTP request.
+    "details": |-
+      This setting can be used to override the server-provided value of `SCRIPT_NAME`, which is most commonly used
+      for hosting Nautobot in a subdirectory (e.g. _example.com/nautobot/_).
+
+      !!! important
+          To host Nautobot under a subdirectory you must set this value to match the same prefix configured on
+          your HTTP server. For example, if you configure NGINX to serve Nautobot at `/nautobot/`, you must
+          set `FORCE_SCRIPT_NAME = "/nautobot/"`.
+    "see_also":
+      "Django documentation for `FORCE_SCRIPT_NAME`": "https://docs.djangoproject.com/en/stable/ref/settings/#force-script-name"
+    "type": "string"
+  "GIT_ROOT":
+    "$ref": "#/definitions/absolute_path"
+    "default": "~/.nautobot/git"
+    "description": >-
+      The file path to a directory where cloned [Git repositories](../../platform-functionality/gitrepository.md)
+      will be located.
+    "environment_variable": "NAUTOBOT_GIT_ROOT"
+  "GRAPHQL_COMPUTED_FIELD_PREFIX":
+    "default": "cpf"
+    "description": "The prefix used for all computed fields in GraphQL. e.g. `my_field` => `cpf_my_field`"
+    "type": "string"
+  "GRAPHQL_CUSTOM_FIELD_PREFIX":
+    "default": "cf"
+    "description": "The prefix used for all custom fields in GraphQL. e.g. `my_field` => `cf_my_field`"
+    "type": "string"
+  "GRAPHQL_RELATIONSHIP_PREFIX":
+    "default": "rel"
+    "description": >-
+      The prefix used for all relationship associations in GraphQL. e.g. `my_relationship` => `rel_my_relationship`.
+    "type": "string"
+  "HTTP_PROXIES":
+    "default": false
```

```suggestion
    "default": ~
```
Tilde is how we're saying None?

---

https://yaml.org/type/null.html - `null` is also valid; I'll change it accordingly for clarity.